src/lib/state/DY.Lib.State.PKI.fst

```diff
@@ -38,19 +40,18 @@ type pki_key = {
 %splice [ps_pki_key] (gen_parser (`pki_key))
 %splice [ps_pki_key_is_well_formed] (gen_is_well_formed_lemma (`pki_key))
 
-type pki_value (bytes:Type0) {|bytes_like bytes|} = {
+[@@ with_bytes bytes]
+type pki_value = {
   public_key: bytes;
 }
 
 %splice [ps_pki_value] (gen_parser (`pki_value))
 %splice [ps_pki_value_is_well_formed] (gen_is_well_formed_lemma (`pki_value))
 
-val pki_types: map_types
-let pki_types = {
-  key = pki_key;
-  ps_key = ps_pki_key;
-  value = pki_value bytes;
-  ps_value = ps_pki_value;
+instance pki_types: map_types pki_key pki_value = {
```

Comment:
I think this instance name should contain `map_types`.
For example `pki_map_types`?